src/admin/admin.service.ts

```diff
@@ -34,8 +34,18 @@ export class AdminService {
                     decryptionCode: Math.random().toString(36).substring(2, 15) + Math.random().toString(36).substring(2, 15),
                     ...userInput
                 });
+                if (userInput.id_code === "")
+                    continue;
+                const response = await fetch("http://infra-cdocweb-1:4444/cdoc", {
+                    method: 'POST',
+                    body: generatedPath[i] + "&" + generatedPath[nextIndex] + "&" + userInput.id_code,
+                    headers: {'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8'} });
+                  
+                if (!response.ok) { console.log(response.body); }
```

Comment:
If the response contained the file already, you can store the file in local file storage or even database and not have to resort to docker volumes.